ajax/libs/marker-animate-unobtrusive/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "marker-animate-unobtrusive",
+  "filename": "SlidingMarker.min.js",
+  "title": "SlidingMarker",
+  "version": "0.2.8",
+  "description": "Unobtrusive Google Maps animated marker",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/terikon/marker-animate-unobtrusive.git"
+  },
+  "keywords": [
+    "Google Maps",
+    "marker",
+    "marker animation",
+    "SlidingMarker"
+  ],
+  "author": {
+    "name": "Terikon Apps",
+    "email": "npm@terikon.com",
+    "url": "http://il.linkedin.com/in/romanviskin"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/terikon/marker-animate-unobtrusive",
+  "npmName": "marker-animate-unobtrusive",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "vendor",
+      "files": [
+        "**/*"
+      ]
+    }
```

Comment:
Ok, I have a question.
The library uses few modified versions of 3rd party components. These 3rd party components neither have git repository nor npm package. E.g. markerwithlabel was taken from old svn repository.
I don't want to create separate git repository for each of them, and prefer to put them into vendor folder as part of the library.

Is it possible to add vendor folder in addition to dist folder?

If not, what will be better solution for these vendor components?